### PR TITLE
Move SSH keys to authorized_keys file editable using Nest Bot

### LINKED ISSUE #53: Improve handling of SSH keys
SSH keys should be able to be updated by Nest Bot. However: Should they stay stored on Authentik, or should the `~/.ssh/authorized_keys` file be treated as the single source of truth?

### PR DESCRIPTION
**Pull request by polypixeldev**
See #53 for more info.

Requires actions on Nest VM:
- Remove `AuthorizedKeysCommand` from `/etc/ssh/sshd_config` (and the associated script `/etc/ssh/ldap_script.sh`)
- Write and run a script to migrate all Nest user's keys from Authentik to an `authorized_keys` file
- Modify `/etc/sudoers` file to allow passwordless execution as root of `/home/nest-internal/nest/nest-bot/src/os/scripts/read_keys.sh` and `/home/nest-internal/nest/nest-bot/src/os/scripts/set_keys.sh`

Note: the migration is just for the sake of proper documentation - it has already been applied in production.

Closes #53

### CODE REVIEW

File: nest-bot/src/os/get_shells.ts
Comment (cskartikey) [inline nest-bot/src/os/get_shells.ts:1]: this should have been it's own PR?

File: nest-bot/src/tests/test.ts
Comment (cskartikey) [inline nest-bot/src/tests/test.ts:1]: is this required? If not, we can remove it.

**Comment by cskartikey** [comment]
Do admins have a way to manage users' ssh keys?

File: nest-bot/src/os/get_shells.ts
Comment (polypixeldev) [reply nest-bot/src/os/get_shells.ts:1]: Just moved it from util dir, github didn't recognize it as rename

File: nest-bot/src/tests/test.ts
Comment (polypixeldev) [reply nest-bot/src/tests/test.ts:1]: Yeah this is from kieran's pr, we can remove it, do you want that in this PR?

**Comment by polypixeldev** [comment]
> Do admins have a way to manage users' ssh keys?

Since SSH keys will just be in `authorized_keys` file, admins can just edit that

File: nest-bot/src/tests/test.ts
Comment (cskartikey) [reply nest-bot/src/tests/test.ts:1]: Preferably not

File: nest-bot/src/os/get_shells.ts
Comment (cskartikey) [reply nest-bot/src/os/get_shells.ts:1]: Feel free to resolve

**Review by cskartikey — APPROVED** [review]
lgtm, feel free to merge.

**Comment by polypixeldev** [comment]
will merge after the upgrade is done

**Review by dispherical — APPROVED** [review]

**Pull request merged**